frontend/src/components/general/Footer.js

```diff
@@ -82,11 +88,17 @@ const SmallFooter = ({ className, noSpacingTop, noAbsolutePosition }) => {
   const classes = useStyles();
   const isNarrowScreen = useMediaQuery(theme => theme.breakpoints.down("xs"));
 
+  const trigger = useScrollTrigger({
```

Comment:
This seems to be initialized as `false` which causes the Footer to show by default if you haven't scrolled yet.
However, the footer shouldn't be shown if you're scrolled all the way to the top, because otherwise, you might not be able to see a lot of projects/organizations/members without scrolling down when you're on the `/browse` page.